src/cmdstan/command.hpp

```diff
@@ -276,7 +276,19 @@ int command(int argc, const char *argv[]) {
 #ifdef STAN_THREADS
   n_threads = get_arg_val<int_argument>(parser, "num_threads");
 #endif
+  // Need to make sure these two ways to set thread # match.
```

Comment:
What happens in line 277 whenever STAN_THREADS is not defined? In that case that line is not there, but the user may correctly specify num_threads=1 or is an error being raised if num_threads is given as argument?

Reply:
I just changed this so that the `num_threads` argument is always equal to 1 if `STAN_THREADS` is not defined. So I think now we should always be fine